R/zchunk_LB125.LC_tot.R

```diff
@@ -65,38 +64,132 @@ module_aglu_LB125.LC_tot_DISABLED <- function(command, ...) {
     # Note that all precursor names (in `add_precursor`) must be in this chunk's inputs
     # There's also a `same_precursors_as(x)` you can use
     # If no precursors (very rare) don't call `add_precursor` at all
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L125.LC_bm2_R %>%
+      add_title("Total land cover by GCAM region") %>%
+      add_units("bm2") %>%
+      add_comments("Aggregated Land area by GCAM region") %>%
+      add_comments("Rounded to 2 digit") %>%
       add_legacy_name("L125.LC_bm2_R") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
+      add_precursors("temp-data-inject/L122.LC_bm2_R_HarvCropLand_Yh_GLU",
+                     "temp-data-inject/L122.LC_bm2_R_OtherArableLand_Yh_GLU",
+                     "temp-data-inject/L123.LC_bm2_R_MgdPast_Yh_GLU",
+                     "temp-data-inject/L123.LC_bm2_R_MgdFor_Yh_GLU",
+                     "temp-data-inject/L124.LC_bm2_R_Shrub_Yh_GLU_adj",
+                     "temp-data-inject/L124.LC_bm2_R_Grass_Yh_GLU_adj",
+                     "temp-data-inject/L124.LC_bm2_R_UnMgdPast_Yh_GLU_adj",
+                     "temp-data-inject/L124.LC_bm2_R_UnMgdFor_Yh_GLU_adj",
+                     "L120.LC_bm2_R_UrbanLand_Yh_GLU",
+                     "L120.LC_bm2_R_Tundra_Yh_GLU",
+                     "L120.LC_bm2_R_RckIceDsrt_Yh_GLU") %>%
       # typical flags, but there are others--see `constants.R`
       add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
       L125.LC_bm2_R
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L125.LC_bm2_R_GLU %>%
+      add_title("Total land cover by GCAM region and GLU") %>%
+      add_units("bm2") %>%
+      add_comments("Aggregated Land area by GCAM region x GLU") %>%
       add_legacy_name("L125.LC_bm2_R_GLU") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
+      add_precursors("temp-data-inject/L122.LC_bm2_R_HarvCropLand_Yh_GLU",
+                     "temp-data-inject/L122.LC_bm2_R_OtherArableLand_Yh_GLU",
+                     "temp-data-inject/L123.LC_bm2_R_MgdPast_Yh_GLU",
+                     "temp-data-inject/L123.LC_bm2_R_MgdFor_Yh_GLU",
+                     "temp-data-inject/L124.LC_bm2_R_Shrub_Yh_GLU_adj",
+                     "temp-data-inject/L124.LC_bm2_R_Grass_Yh_GLU_adj",
+                     "temp-data-inject/L124.LC_bm2_R_UnMgdPast_Yh_GLU_adj",
+                     "temp-data-inject/L124.LC_bm2_R_UnMgdFor_Yh_GLU_adj",
+                     "L120.LC_bm2_R_UrbanLand_Yh_GLU",
+                     "L120.LC_bm2_R_Tundra_Yh_GLU",
+                     "L120.LC_bm2_R_RckIceDsrt_Yh_GLU") %>%
       # typical flags, but there are others--see `constants.R`
```

Comment:
Oh the flags should absolutely be kept. but the comment "# typical flags, but there are others..." is no longer relevant